### ISSUE TITLE
Random crashing along with false (?) low memory warning

### ISSUE THREAD

**Issue by anticobalt** [Kaldaien/FAR#67]
https://pastebin.com/pMt0mnY9
https://pastebin.com/aw1XVu6z
https://pastebin.com/DKgRPfMc

Apologizes if this isn't the right place to ask for help, or if this a problem with the game itself and not the mod.

The crashing seem to happen whenever, whether I'm playing or changing the graphic settings. Windows 7 throws a low memory alert, and most of the time this causes the game (and other programs I have open) to crash, despite the fact that I'm literally monitoring the RAM usage with task manager and it's not going above 70% of the 8GB I have. I did reinstall my graphics card driver, but that didn't seem to help. I already have both of the updates suggested for Windows 7 users on the steam discussion page installed. The only interfering program I have is Discord, and I did try playing the game with it exited, but no changes.

I am using a PS3 controller (via SCPToolkit) and Steam Big Picture mode.

The log files linked are from right after the game crashed after I tried to switch from fullscreen to windowed mode in the main menu settings.

**Comment by Idearum** [comment]
Stuff that hooks into your game that might be at fault:
- Listary
- DisplayFusion
- Some Bluetooth software? O.o

The low memory warning, does it complain about system RAM or graphics VRAM ? Windows 7 throws a warning at around 75% of VRAM in use. This warning can't be disabled at all, regardless of what you might've read online (a bunch of "fixes" are listed for it, none of which works).

**Comment by anticobalt** [comment]
It would simply say that the computer "is low on memory", and ask that I exit the game, so I guess that's just system RAM.

Anyways, I disabled the three programs you mentioned, and that seems to have solved the crashing. Couldn't get Steam to recognize the controller without SCPToolkit, but it doesn't seem to be causing any problems. I've played the game for about 5 hours and it only crashed once (right after the intro mission, when I tried to change the graphics settings from medium to high).

I was still getting a memory warning, but it was a more generic one this time (https://vgy.me/YGsocn.png). Turns out I was running out of "commit memory" (https://vgy.me/5JL0DT.png) while the game was running, so I increased the pagefile.sys size, and that seemed to fix it. The game, according to Resource Monitor, was demanding around 8GB of commit memory.

Thanks for the help! Hopefully the game runs smoothly from here on out.
